src/lib.rs

```diff
@@ -190,36 +171,40 @@ impl PublicKeys {
     pub fn encrypt_anonymous_bytes(&self, plaintext: &[u8]) -> Vec<u8> {
```

Comment:
I think `anonymously_encrypt()` and `anonymously_encrypt_bytes()` are clearer function names, but maybe confirm with @ustulation before making this change if you agree?